client/src/components/drawer/AdminList.tsx

```diff
@@ -63,6 +63,12 @@ export default function AdminList() {
             icon={<NetworkCheck />}
             feature={Features.FORM}
           />
+          <DrawerEntry
+            title="Audit Log"
+            to="/audit"
+            icon={<LibraryBooks />}
```

Comment:
I think this icon should be our activity entry icon, I feel like it doesn't really spell security as much and is far too similar to the activity entry icon

What about this icon:

![image](https://user-images.githubusercontent.com/11084692/198185312-fec36dd7-693f-4057-b75e-fb5b8a626bfa.png)

Or this one:

![image](https://user-images.githubusercontent.com/11084692/198185498-6de4ba7b-3734-48bc-b6a5-3a3bc3c795fd.png)

as alternatives?

Reply:
hehe RECENT ACTORS a perfect name for the biggest actors